tasmota/xsns_47_max31865.ino

```diff
@@ -1,7 +1,7 @@
 /*
   xsns_39_MAX31865.ino - MAX31865 thermocouple sensor support for Tasmota
 
-  Copyright (C) 2021  Alberto Lopez Siemens
+  Copyright (C) 2020 Alberto Lopez Siemens
```

Comment:
Did you rebase since beginning of this year? I updated this a month ago.

Reply:
true, sorry - had a merge conflict and forgot the header comment to accept. All changes are based on #10676